src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200103101609_AddReleaseUpsertStoredProcs.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System.IO;
+using System.Reflection;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class AddReleaseUpsertStoredProcs : Migration
+    {
+        private const string MigrationsPath = "Migrations";
+        private const string MigrationId = "20200103101609";
+
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Types
+            ExecuteFile(migrationBuilder, $"{MigrationId}_TableTypes.sql");
+            // Routines
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilter.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilterFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilterGroup.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilterGroupFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilterItem.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFilterItemFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertIndicator.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertIndicatorFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertIndicatorGroup.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertLocation.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertPublication.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertRelease.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertSubject.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertSubjectFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertTheme.sql");
+            ExecuteFile(migrationBuilder, $"{MigrationId}_Routine_UpsertTopic.sql");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            // Routines
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilter.sql");
```

Comment:
I will correct these because the drop procedure statements shouldn't have `.sql` on the end of them.